js/views/modals/purchase/Purchase.js

```diff
@@ -252,10 +252,18 @@ export default class extends BaseModal {
           contentType: 'application/json',
         })
           .done((data) => {
-            this.updatePageState('pending');
             this.actionBtn.render();
             this.purchase.set(this.purchase.parse(data));
-            this.payment.render();
+            this.payment = this.createChild(Payment, {
```

Comment:
Since you're only going to post ob/purchase once, I don't think it's necessary. Do you see any possible scenario where that guy would be posted more than once?